README.md

```diff
@@ -77,7 +77,9 @@ Load and prepare data by running the following code:
 
 This dataset is created by public data collected by [crawler scripts](scripts/data_collector/), which have been released in
 the same repository.
-Users could create the same dataset with it.
+Users could create the same dataset with it. 
+
+*Please pay **ATTENTION** that the data is collected from [Yahoo Finance](https://finance.yahoo.com/lookup) and the data might not be perfect. We recommend users to prepare their own data if they have high-quality dataset. For more information, users can refer to the [related document](https://qlib.readthedocs.io/en/latest/component/data.html)*
```

Comment:
[This link](https://qlib.readthedocs.io/en/latest/component/data.html#converting-csv-format-into-qlib-format) will be more accurate

Reply:
OK, I will fix this.